Integrations/SyslogSender/SyslogSender.py

```diff
@@ -0,0 +1,278 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+''' IMPORTS '''
+
+from contextlib import contextmanager
+from logging.handlers import SysLogHandler
+from distutils.util import strtobool
+import requests
+import logging
+import socket
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+''' CONSTANTS '''
+
+
+PLAYGROUND_INVESTIGATION_TYPE = 9
+INCIDENT_OPENED = 'incidentOpened'
+LOGGING_LEVEL_DICT = {
+    'INFO': logging.INFO,
+    'DEBUG': logging.DEBUG,
+    'WARNING': logging.WARNING,
+    'ERROR': logging.ERROR,
+    'CRITICAL': logging.CRITICAL
+}
+SEVERITY_DICT = {
+    'Unknown': 0,
+    'Low': 1,
+    'Medium': 2,
+    'High': 3,
+    'Critical': 4
+}
+
+''' Syslog Manager '''
+
+
+class SyslogManager:
+    def __init__(self, address: str, port: int, protocol: str, logging_level: int):
+        """
+        Class for managing instances of a syslog logger.
+        :param address: The IP address of the syslog server.
+        :param port: The port of the syslog server.
+        :param protocol: The messaging protocol (TCP / UDP).
+        :param logging_level: The logging level.
+        """
+        self.address = address
+        self.port = port
+        self.protocol = protocol
+        self.logging_level = logging_level
+
+    @contextmanager  # type: ignore[misc, arg-type]
+    def get_logger(self) -> logging.Logger:
+        """
+        Get a new instance of a syslog logger.
+        :return: syslog logger
+        """
+        handler = self._get_handler()
+        syslog_logger = self._init_logger(handler)
+        try:
+            yield syslog_logger
+        finally:
+            syslog_logger.removeHandler(handler)
+            handler.close()
+
+    def _get_handler(self) -> SysLogHandler:
+        """
+        Get a syslog handler for a logger according to provided parameters.
+        :return: syslog handler
+        """
+        if self.protocol == 'tcp':
+            return SysLogHandler((self.address, self.port), socktype=socket.SOCK_STREAM)
+        else:
+            return SysLogHandler((self.address, self.port))
```

Comment:
would be more elegant to initialize one instance of `SysLogHandler`, so lets build the args passed to it (hostname, port and socktype) and then send using the `**` operator